client/cmd/up_test.go

```diff
@@ -36,7 +36,7 @@ func TestUp_Start(t *testing.T) {
 
 func TestUp(t *testing.T) {
 
-	defer iface.Close()
+	defer iface.Close("wt0")
```

Comment:
This value will be ignore for Mac and Windows.
Mac and Windows use tun shared variable of a userspace go.

This package has to be rewritten I think because it is misleading a bit